### DISCUSSION TITLE
Dependabot check in github actions

### DISCUSSION THREAD

**Discussion by roisec** [community/community#12345] [Code Security]
Hi,
I tried to figure out how we can block build by dependabot with Github actions.
The goal is to scan security vulnerabilities with dependabot and block merging.
Dependabot support this?

**Comment by Lovleyangel90** [comment]
Helping find that make sense for you ?

**Comment by exvuma** [comment]
👋🏼 Dependabot security updates work by opening a pull request once a security alert is found. Dependabot does not actively scan your code but rather actively updates your code. Once you are set up using Dependabot security updates, your code will be up to date until a new vulnerability is found.

To actively prevent dependencies from entering your codebase, you can check out this new Github Action: [Dependency Review Action
](https://github.com/dsp-testing/dependency-review-action)

**Comment by HariHara-sn** [comment]
your resumes, seems good
